src/interactive_markers/interactive_marker_server.py

```diff
@@ -382,6 +382,13 @@ def publishInit(self):
 
             self.init_pub.publish(init)
 
+    # return all interactive markers
+    def getNames(self):
+        names = []
+        for marker_context in self.marker_contexts.items():
+            names.append(marker_context.int_marker.name)
+        return names
```

Comment:
I suggest we use a list comprehension. Also, I think we should use `keys()` (instead of `items()`) to get the name:

```suggestion
        return [marker_name for marker_name in self.marker_contexts.keys()]
```